contentcuration/contentcuration/frontend/channelList/views/ChannelSet/ChannelSetModal.vue

```diff
@@ -433,6 +422,7 @@
         }
       },
       confirmCancel() {
+        this.showUnsavedDialog = false;
```

Comment:
`showUnsavedDialog` is set to false within `this.close()`, which is called in this function already, so we don't need this line added here.